manifests/charts/dash/values.yaml

```diff
@@ -43,6 +43,38 @@ service:
 
 prometheusOperator:
   enabled: true
```

Comment:
Should we be defaulting this to true? If they don't have kube-prometheus-operator installed are we going to end up deploying anything that will cause m9sweeper not to install?

Reply:
@jason-woodman @jasonWoodman  Probably a question for you ^^^

Reply:
Becky is going to set it to enabled:false and then we can merge.